params.go

```diff
@@ -124,6 +124,11 @@ type Params struct {
 	// Account is deprecated form of StripeAccount that will do the same thing.
 	// Please use StripeAccount instead.
 	Account string
+
+    // StripeVersion sets the version for the API to use when parsing requests
+    // and rendering responses.  This string refers to a date, such as
+    // "2017-05-25"
+    StripeVersion string
```

Comment:
Ugh, so this is going to make implementation a little more awkward, but we shouldn't add a top-level `StripeVersion` here.

stripe-go ties itself to a particular version of the Stripe API, and this is probably something that we want to eventually emulate from every library. Even if that doesn't happen though, we should make sure that this continues to be the case for stripe-go because otherwise this makes versioning available on every endpoint and makes people's use of the library much less deterministic for us.

I think we'll have to move this down to the `EphemeralKeyParams` and figure out another way to inject it into the request headers.